setup.py

```diff
@@ -49,7 +49,8 @@
     zip_safe=False,
     platforms='any',
     install_requires=[
-        'Invenio>=1.9999.4',
+        'Invenio>=2.1',
```

Comment:
we should not depend on `Invenio` itself, but rather `invenio-base`, `invenio-ext`, etc.